overlord/snapstate/backend/copydata_test.go

```diff
@@ -76,7 +76,7 @@ func (s *copydataSuite) TestCopyData(c *C) {
 
 	v1 := snaptest.MockSnap(c, helloYaml1, &snap.SideInfo{Revision: snap.R(10)})
 	// just creates data dirs in this case
-	err = s.be.CopySnapData(v1, nil, progress.Null)
+	err = s.be.CopySnapData(v1, nil, progress.Null, nil)
```

Comment:
same considerations here in backend about testing with options not nil and the flag set?